feature-libs/customer-ticketing/core/facade/customer-ticketing.service.spec.ts

```diff
@@ -93,4 +93,62 @@ describe('CustomerTicketingService', () => {
         });
     });
   });
+  describe('getTicketCategories', () => {
```

Comment:
nice 👍

Reply:
These tests are actually failing

have you ran `ng test customer-tickerting`. I said nice above because I saw tests, but they're not passing. 
What needs to be done is that it requires the mock connector to be aware of these functions, so include them here https://github.com/SAP/spartacus/pull/16142/files#diff-450d65a6cbacd13bee796c96a548940c8ee202110c969f6fe9e25a335fda26ddR32